exampleCourse/questions/demo/workspace/rstudio/question.html

```diff
@@ -10,13 +10,12 @@
 - a numeric `nn` with value 3.1415
 - a text variable `cc` with text `hello, world!` including the comma an exclamation
 
-You can press the button below to launch a demo RStudio workspace.
+You can press the button below to launch a demo RStudio workspace. The file must be
+completed in the workspace.
```

Comment:
```suggestion
Press the button below to launch an RStudio workspace. The object above must be created in the `student.R` file inside the workspace.
```

I'm ok with other ways to phrase it, but (1) it needs to be clearer that opening the workspace is required, and (2) the file name is important in this context.

Reply:
I like this wording a bit more.